### DISCUSSION TITLE
More customisability for USER experience

### DISCUSSION THREAD

**Discussion by MrDirkelz** [elementary/triage#667] [Ideas]
### Minimise Button
I really like elementary so far. But where is the minimise button, I actually laughed when I couldn't minimise, it's just such a basic quality of life in ALL operating systems. 

### Overall quality of life features
What about making a built in version of **pantheon-tweaks**?? Currently I am missing a lot of features I could just find in a Gnome or KDE enviroment, maybe try to see what they did and implement it? This distro is so close to being 100% what I want, and I believe most other elementary guests want. Why would I rather opt for elementary, than say Ubuntu and just customise it to look like pantheon? An operating system is supposed to belong to the user using it isn't it? 

Why not make the dock more customisable? I prefer the macOs type dock, but other users may not. Some users might prefer it on other spaces on their screens or to fill the screen, or maybe even a different color than those available to fit their wallpapers?

### Other ideas that would be nice, but not neccesary.
A frosted glass effect available to turn on, on the window manager, and the dock, and the panel. 
The button in the multitask view to add a new workspace doesn't look modern. It's fine, but doesn't really fit the overall vibe of the new updates coming to operating systems. 

### Oh and what's up with the app store having almost none of the apps I want but I can install them via _**apt**_?

**Comment by teamcons** [comment]
Hello
You can minimize. The shortcut is there, the menu option is there, and you can enable the button easily.
It isnt there by default because it isnt the intention.
Gnome does not provide minimize by default either 

Tweaks is a third party project. It is a click away. Not everyone want to tweak their system. Nobody stops you - its your system.

The dock is brand new. Ive built a theme switcher and it allows for transparency, as third party project. But the dev team is focused in making a dock for eOS, working well with the default workflow, not a dock that covers everything everyone ever could want - and that is extremely reasonable. Dev time is precious.
There is cairo-dock which tries to fill all purposes. You are free to install it and remove the default one.

The appstore is flatpak only. That is the intended supported format. But you can use apt just fine in the terminal.

The system is still being worked on. Like the button to add workspaces. eOS has limited resources - you are however welcome to work with the team to share design, participate in discussions and submit pull requests

**Reply by MrDirkelz** [reply]
Hi! Thanks for the reply!

Yes I realised that after starting to use gnome. But I feel eOS could potentially maybe change the linux landscape in the future.
I am referring to people who have never used a linux desktop before and don't want Apple or Mac to decide what is good and what is not. Yes there are ways to tweak it to your liking if you're OCD about how your desktop looks and feel, but why is it that some linux distros are moving away from making their OS usable for somebody whom has never before used a computer for example, because not everyone wants to use the terminal.

All your points are totally fair and I agree, I'm just seeing things from the other end of the tunnel.

I would like to participate but I don't have the tech skills or design skills for that yet, I have only been studying and working on Web based projects for a year now. But yeah, I hope eOS finds some resources and funding to make this OS the next big thing for linux.

This is the first linux OS that I actually cared enough about to actually make a "review"/"comment" on, because well yeah, it seems like it has the potential to come out on top

**Reply by teamcons** [reply]
Hi !
I hope i did not sound grating??

> Yes I realised that after starting to use gnome. But I feel eOS could potentially maybe change the linux landscape in the future. I am referring to people who have never used a linux desktop before and don't want Apple or Mac to decide what is good and what is not. Yes there are ways to tweak it to your liking if you're OCD about how your desktop looks and feel, but why is it that some linux distros are moving away from making their OS usable for somebody whom has never before used a computer for example, because not everyone wants to use the terminal.

It much depends on the distribution and what experience they want to do.

Linux Mint, for example, is all about a traditional, pack-it-all experience... Thats one approach.
I personally think, from trying mint, the tech it relies on is old, and it tries too much to cater to as much as possible - ending a fair bit disjointed and a fair bit messy. Though it is still a good option !

elementary OS goes for a more focused approach to support - It is more opinionated.
Which also indeed draws (understandable) critic from people who want it to also offer more than what it wants to do.

> I would like to participate but I don't have the tech skills or design skills for that yet, I have only been studying and working on Web based projects for a year now. But yeah, I hope eOS finds some resources and funding to make this OS the next big thing for linux.

With another we are working on a documentation app. The documentation is done in HTML ! We want it to explains the basics of how the OS is thought, and present its elements.
A lot of people have eOS because they are not tech literate and someone installed it for them.
I want to give people a comprehensive, always present help that way ! Once it is solid enough we will present it to be a default app.
Do you want to help us ?

Else, translation always could need a hand ! eOS has a Weblate. You can suggest and review translations for what you want
 
> This is the first linux OS that I actually cared enough about to actually make a "review"/"comment" on, because well yeah, it seems like it has the potential to come out on top

It is a niche in a land of niches... I think it is important to be honest that no distro can do a one-size-fits-all. I personally like it, and contribute to it - because well it fits my needs ! It runs a solid base, and is pretty and clean out of the box.
But it could indeed be more known and have more helping hands !

**Reply by MrDirkelz** [reply]
> Hi !
> I hope i did not sound grating??
Not at all, you're also just passionate about the project :)
> 
> > Yes I realised that after starting to use gnome. But I feel eOS could potentially maybe change the linux landscape in the future. I am referring to people who have never used a linux desktop before and don't want Apple or Mac to decide what is good and what is not. Yes there are ways to tweak it to your liking if you're OCD about how your desktop looks and feel, but why is it that some linux distros are moving away from making their OS usable for somebody whom has never before used a computer for example, because not everyone wants to use the terminal.
> 
> It much depends on the distribution and what experience they want to do.
> 
> Linux Mint, for example, is all about a traditional, pack-it-all experience... Thats one approach.
> I personally think, from trying mint, the tech it relies on is old, and it tries too much to cater to as much as possible - ending a fair bit disjointed and a fair bit messy. Though it is still a good option !
> 
> elementary OS goes for a more focused approach to support - It is more opinionated.
> Which also indeed draws (understandable) critic from people who want it to also offer more than what it wants to do.
Yeah that is a very good point and I agree, having a solid goal and idea set in place helps development go smoother, and besides other features can always be added as extensions that users can make dedicated to eOS, which could also make it more of an enriching enviroment. So that the main developers can focus on what needs to be focused on
> 
> > I would like to participate but I don't have the tech skills or design skills for that yet, I have only been studying and working on Web based projects for a year now. But yeah, I hope eOS finds some resources and funding to make this OS the next big thing for linux.
> 
> With another we are working on a documentation app. The documentation is done in HTML ! We want it to explains the basics of how the OS is thought, and present its elements.
> A lot of people have eOS because they are not tech literate and someone installed it for them.
> I want to give people a comprehensive, always present help that way ! Once it is solid enough we will present it to be a default app.
> Do you want to help us ?
Yeah, maybe I can see how it works and how it blends with my schedule, it is something that lots of linux distros lack, they mostly just have ugly 2000's style wikis from what i saw, especially distros like linux mint
> 
> Else, translation always could need a hand ! eOS has a Weblate. You can suggest and review translations for what you want
>  
> > This is the first linux OS that I actually cared enough about to actually make a "review"/"comment" on, because well yeah, it seems like it has the potential to come out on top
> 
> It is a niche in a land of niches... I think it is important to be honest that no distro can do a one-size-fits-all. I personally like it, and contribute to it - because well it fits my needs ! It runs a solid base, and is pretty and clean out of the box.
> But it could indeed be more known and have more helping hands !
> 
Yes indeed! And if I can help on the web side of things I will gladly, I have some experience in more than html as well, like javascript, Front end frameworks, backend, etc. 

I wrote this on a phone and rarely use github for discussions so excuse my markdown noobishness

**Comment by svandragt** [comment]
Alternatively, you minimise by clicking on the dock icon for the application you want to minimize or press a shortcut. But hiding the window is more elegant.

**Reply by teamcons** [reply]
clicking on the dock focuses the window or shows a switcher for all open windows

hiding the window is minimizing.

**Reply by svandragt** [reply]
Ah, thanks, I'm on elementaryOS 7 still, maybe that behaviour changed or I customised it.
